01_login.php

```diff
@@ -53,17 +52,40 @@
 		<!-- internal style sheet -->
 		<style>
 			h1 {
-				font-family: 'Alegreya';font-size: 35px;
+				font-family: 'Helvetica', 'Arial', sans-serif;
```

Comment:
new font + size added: Helvetica is very easy to read and understand and makes the website more professional looking